ote/src/cljs/ote/views/transit_changes.cljs

```diff
@@ -190,19 +190,41 @@
                                       "Ei löydettyjä muutoksia")
                    :name->label str
                    :label-style (merge style-base/table-col-style-wrap {:font-weight "bold"})
-                   :stripedRows true
-                   :row-style {:cursor "pointer"}
-                   :show-row-hover? true
-                   :on-select (fn [evt]
-                                (let [{:keys [transport-service-id date]} (first evt)]
-                                  (e! (tc/->ShowChangesForService transport-service-id
-                                                                  date))))}
+                   :stripedRows true}
```

Comment:
Clicking a row would be much sharper UX instead of clicking links.  Is the new UX just as good? 
To see the row URL in previous implementation, alternatively a tooltip could have been used.

Reply:
Agreed. I'll do some changes.